lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -1036,8 +1036,11 @@ def install_puppetlabs_release_repo_on( hosts, repo = nil, opts = options )
               # package. We'll have to remember to update this block when
               # we update the signing keys
               if variant == 'sles' && version >= '11'
-                on host, "wget -O /tmp/puppet-gpg-key http://yum.puppetlabs.com/RPM-GPG-KEY-puppet"
-                on host, "rpm --import /tmp/puppet-gpg-key"
+                %w[puppet puppet-20250406].each do |gpg_key|
+                  on host, "wget -O /tmp/#{gpg_key} http://yum.puppetlabs.com/RPM-GPG-KEY-#{gpg_key}"
+                  on host, "rpm --import /tmp/#{gpg_key}"
+                  on host, "rm -f /tmp/#{gpg_key}"
```

Comment:
Normally I'd suggest to avoid the temporary file altogether by using `rpm --import <(wget -O - $URL)` but I don't know if it works here.

Reply:
Looks like this doesn't work as expected (tested on SLES 12):

```
tragic-yarn:~ # rpm --import <(wget -q -O - https://yum.puppet.com/RPM-GPG-KEY-puppet)
error: /dev/fd/63: import read failed(0).
```

Reply:
`rpmkeys --import` can take a URL directly on Fedora, but just gives `import read failed(2)` on openSUSE Tumbleweed for some reason, even though it's using the same version of `rpm`. :man_shrugging: